frontend/src/features/admin-form/create/builder-and-design/useDesignStore.tsx

```diff
@@ -9,7 +9,7 @@ import { UploadedImage } from './BuilderAndDesignDrawer/EditFieldDrawer/edit-fie
 export type CustomLogoMeta = Omit<CustomFormLogo, keyof FormLogoBase>
 
 export enum DesignState {
-  EditingHeader,
+  EditingHeader = 1,
```

Comment:
why is this `=1` needed?

Reply:
We want the enum to start from `1` (so falsey footguns, like when value is `0` and you do a negate check is less likely to happen)